src/app/technical-record/body/edit/body-edit.component.html

```diff
@@ -0,0 +1,68 @@
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-full">
+    <div class="govuk-form-group">
+      <fieldset class="govuk-fieldset">
+
+        <form [formGroup]="techRecordFg">
+
+          <div class="govuk-form-group">
+            <label class="govuk-label govuk-!-font-weight-bold" for="make">
+              Make
+            </label>
+            <input id="make" formControlName="make" type="text" class="govuk-input govuk-input--width-20" />
+          </div>
```

Comment:
we should add the <select> options

Reply:
I think the design is wrong, i checked the swaggerfile and it is shown as a string i believe
"make": "string",